invenio_assets/filters.py

```diff
@@ -85,3 +102,36 @@ def output(self, _in, out, **kw):
     def input(self, _in, out, **kw):
         """Input filtering."""
         self.subprocess([self.binary or 'cleancss'], out, _in)
+
+
+class AngularGettextFilter(Filter):
+    """Compile GNU gettext messages to angular-gettext module."""
+
+    name = 'angular-gettext'
+
+    options = {
+        'catalog_name': None,
+    }
+
+    def output(self, _in, out, **kwargs):
+        """Wrap translation in Angular module."""
+        out.write(
+            'angular.module("{0}", ["gettext"]).run('
+            '["gettextCatalog", function (gettextCatalog) {{'.format(
+                self.catalog_name
+            )
+        )
+        out.write(_in.read())
+        out.write('}]);')
+
+    def input(self, _in, out, **kwargs):
+        """Process individual translation file."""
+        catalog = read_po(_in)
+        out.write('gettextCatalog.setStrings("{0}", '.format(
+            catalog.language_team.split(' ')[0]
```

Comment:
Did you manage to find a way to extract the language rather than getting it from the language team?